package.json

```diff
@@ -7,7 +7,8 @@
     "Remoting",
     "REST"
   ],
-  "version": "2.24.0",
+  "version": "3.0.0-alpha.1",
+  "publishConfig": { "tag": "next" },
```

Comment:
In LoopBack, we actually use spaces inside curly brackets, as that's the style used in Node core. `{ "tag": "next" }`.

However, in this particular case, it's better to put the contents on new line, because that's what npm will do the next time somebody runs `npm install --save`.

```json
{
  "publishConfig": {
    "tag": "next"
  }
}
```